get_hashsets.js

```diff
@@ -9,7 +9,7 @@ const request = require("request");
 const pkg = require("./package.json");
 
 const HIBP_AUTH = `Bearer ${AppConstants.HIBP_API_TOKEN}`;
-const USER_AGENT = `${pkg.name}@${pkg.version}`;
+const HIBP_USER_AGENT = `${pkg.name}/${pkg.version}`;
```

Comment:
Renamed for consistency (since it is HIPB-specific), and tweaked to use a `/` separator instead of `@` (since a backslash seems to be more of a standard).